src/sage/geometry/polyhedron/combinatorial_polyhedron/base.pyx

```diff
@@ -747,9 +748,9 @@ cdef class CombinatorialPolyhedron(SageObject):
 
         EXAMPLES::
 
-            sage: P = polytopes.permutahedron(3)                                        # optional - sage.combinat
-            sage: C = CombinatorialPolyhedron(P)                                        # optional - sage.combinat
-            sage: C.Hrepresentation()                                                   # optional - sage.combinat
+            sage: P = polytopes.permutahedron(3)
+            sage: C = CombinatorialPolyhedron(P)
+            sage: C.Hrepresentation()
             (An inequality (1, 1, 0) x - 3 >= 0,
```

Comment:
This is a real fix of misplaced tags?

Reply:
Yes, this `# optional` is no longer needed, probably because I lowered `sage.combinat.permutation` to sagemath-categories

Reply:
OK.